README.md

```diff
@@ -33,7 +44,7 @@ as a Nuget package or you can download the source using the instructions above.
 
 ### Prerequisites
```

Comment:
Updated to "
## Managed Windows Client SDK

Our managed portable library for Windows 8, Windows Phone 8, Windows Phone 8.1, and Windows Universal Client SDK makes it incredibly easy to use Mobile Services from your Windows applications."